pages/common/typeorm.md

```diff
@@ -0,0 +1,36 @@
+# typeorm
+
+> A JavaScript ORM that can run in Node.js, browsers, Cordova, Ionic, React Native, NativeScript, and Electron platforms.
+> More information: <https://typeorm.io/>.
+
+- Generate initial TypeORM project structure:
+
+`typeorm init`
+
+- Create an empty migration file:
+
+`typeorm migration:create --name {{migration_name}}`
+
+- Create a migration file with the SQL sentences to update the schema:
+
+`typeorm migration:generate --name {{migration_name}}`
+
+- Run all pending migrations:
+
+`typeorm migration:run`
+
+- Create a new entity file into a specific directory:
+
+`typeorm entity:create --name {{entity}} --dir {{path/to/directory}}`
+
+- Display the SQL sentences to be executed by `typeorm schema:sync` on the default connection:
```

Comment:
Again with the SQL sentences. Is this some kind of tool-specific technical language? 'cause I certainly haven't heard of it elsewhere.